examples/with-styled-components/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { BladeProvider, Button } from '@razorpay/blade/components';
+import { paymentTheme } from '@razorpay/blade/tokens';
+import { ThemeProvider } from 'styled-components';
+import '@fontsource/lato/400.css';
+import '@fontsource/lato/400-italic.css';
+import '@fontsource/lato/700.css';
+import '@fontsource/lato/700-italic.css';
+
+import './app.css';
+
+const customTheme = {
+  colors: {
```

Comment:
aren't we showing how to avoid these conflicts? this is actually the bug wala scenario right?

Reply:
Yup, earlier we were thinking of adding theme key to blade new. Now this repro isn't needed since we're fixing on blade-old. This example needs to be updated to provide a theme key into this `customTheme` object.

Reply:
Added in https://github.com/razorpay/blade/pull/889/commits/75177f0affe2fd32564bd678e2532571c57f3303